reana_cluster/cli/cluster.py

```diff
@@ -68,6 +69,34 @@ def get(ctx, component, namespace):
         logging.debug(str(e))
 
 
+@click.command(help='Configure your shell environment with REANA components'
+                    ' URLs.')
+@click.option(
+    '--namespace',
+    default='default',
+    help='Namespace of the components which configuration shoubd be produced.')
+@click.pass_context
+def env(ctx, namespace):
+    """Produce shell exportable list of REANA components urls."""
+    try:
+        export_lines = []
+        component_export_line = 'export {env_variable_name}={component_url}'
+        for component in reana_env_exportable_components:
+            component_info = ctx.obj.backend.get_component(
+                component, namespace)
+            export_lines.append(component_export_line.format(
+                env_variable_name='{0}_URL'.format(
+                    component.upper().replace('-', '_')),
+                component_url='{schema}://{ip}:{port}'.format(
+                    schema='http',
+                    ip=component_info['external_ip_s'][0],
+                    port=component_info['ports'][0]),
+            ))
+        click.echo('\n'.join(export_lines))
```

Comment:
Note that we still have leading verbose info lines:

```console
$ reana-cluster env 
[INFO] Validating REANA cluster specification file: /home/simko/.virtualenvs/reana-cluster/lib/python3.6/site-packages/reana_cluster/configurations/reana-cluster.yaml
[INFO] /home/simko/.virtualenvs/reana-cluster/lib/python3.6/site-packages/reana_cluster/configurations/reana-cluster.yaml is a valid REANA cluster specification.
[INFO] Cluster type specified in cluster specifications file is 'kubernetes'
export REANA_SERVER_URL=http://192.168.39.211:31331
```

This means that people wouldn't be able to use `eval`...

Could you silence the info lines as part of the same PR?

Reply:
🚀 done. We need to update docs all over the place (i.e. https://github.com/reanahub/reana-demo-helloworld#run-the-example-on-reana-cloud)

Reply:
Thanks,  can you please also:

1. add the command to the "Getting started" guide and to the "User guide".
2. amend the help message of `reana-cluster env --help` as suggested earlier *(hmm, cannot find it in the history, it looks like I forgot to submit my comment? I'll add it below...)*

Reply:
...OK, so the comment was that "Configure your shell environment with REANA components' URLs" might be better said "Display the commands to set up the environment for the REANA client".  The latter would be more understandabale to the users. The reason is that the `env` command does not configure anything, only prints instructions how to do so.  (I'm basically emulating what `docker-machine env` prints for help, which I find nicely descriptive...)